src/views/Home.vue

```diff
@@ -11,12 +11,26 @@
 			<div style="height: 1em"/>
 
 			<h3>Jul 24-25, 2021 • Virtual Event</h3>
-			<!-- <input 
-				type="text"
-				placeholder="Enter your email for updates!"
-				v-model="email"
-				@keyup.enter="addMailList"
-			> -->
+			
+			<!-- Begin Mailchimp Signup Form -->
+			<form action="https://cutc.us10.list-manage.com/subscribe/post?u=73d566326ae6bb629b42fb5fd&amp;id=e28a4a51fe" method="post" 
+					target="_blank" novalidate>
```

Comment:
Why do we have `novalidate`? I guess if we remove this we wouldn't need to do the regex stuff to verify the email, is there anything wrong with that?

Reply:
I think if we used the OG form validation the JS function wouldn't have any way of knowing that the validation failed, so it would transition as if the email was successful

Reply:
Could `form.checkValidity()` here be used instead, as described here? https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/HTML5/Constraint_validation#constraint_validation_process